faker/providers/address/fr_FR/__init__.py

```diff
@@ -467,3 +466,12 @@ def department_number(self) -> str:
         :example: '59'
         """
         return self.department()[0]
+
+    def postcode(self) -> str:
+        """
+        Randomly returns a postcode generated from existing french depertment number.
+
+        exemple: '33260'
+        """
+        department = self.department()
+        return department + self.random_number(digits=5 - len(department))
```

Comment:
if we're concatenating strings, I'd rather use string formatting (f-strings are fine). Just to make it more evident.

Reply:
Done !